src/screens/Customers/CustomerProfile.tsx

```diff
@@ -53,7 +53,12 @@ interface CustomerProps extends RouteComponentProps {
 }
 
 function CustomerProfile(props: CustomerProps) {
-  const { classes, match, currentSite, customer } = props;
+  const { classes, match, currentSite } = props;
+  const customer : CustomerRecord = useSelector(selectCurrentCustomer) || EMPTY_CUSTOMER;
+
+  if (customer === EMPTY_CUSTOMER) {
```

Comment:
For inventory, I ended up taking out the `|| EMPTY` and just had the redirect if condition be `if (!customer)` which I think makes more sense since there's really no good reason for there to be an empty customer except to satisfy the type (which you can just remove).